src/EbmlString.cpp

```diff
@@ -63,7 +63,7 @@ filepos_t EbmlString::ReadData(IOCallback & input, ScopeMode ReadFully)
 
   } else {
     Value.resize(GetSize());
-    std::memset(&Value[0], 0, GetSize());
+    std::fill_n(Value.begin(), GetSize(), 0);
```

Comment:
This line can be removed completely since the next line is supposed to fill the same memory area anyway. And I highly doubt `fill_n` is more efficient than `memset`.

There's also a `std::string` constructor with a size and an init character. (2) in https://en.cppreference.com/w/cpp/string/basic_string/basic_string

Reply:
fill_n is identical to memset.